docs/application/native/guides/multimedia/media-controller.md

```diff
@@ -269,18 +540,19 @@ To retrieve the playlist and metadata information on the client side:
 > This feature supports Tizen 4.0 and Higher for Mobile.
 
 
-## Sending and Processing Commands to Receive Replies
+## Sending and Processing commands
```

Comment:
@haejeongkim 
Change: 
## Sending and Processing commands

to 

## Sending and Processing Commands

Reply:
Done